include/openssl/core_numbers.h

```diff
@@ -415,6 +415,12 @@ OSSL_CORE_MAKE_FUNC(int, OP_keymgmt_get_key_params,
                     (void *key, OSSL_PARAM params[]))
 OSSL_CORE_MAKE_FUNC(const OSSL_PARAM *, OP_keymgmt_gettable_key_params, (void))
 
+#define OSSL_FUNC_KEYMGMT_SET_KEY_PARAMS            29
+#define OSSL_FUNC_KEYMGMT_SETTABLE_KEY_PARAMS       30
+OSSL_CORE_MAKE_FUNC(int, OP_keymgmt_set_key_params,
+                    (void *key, const OSSL_PARAM params[]))
+OSSL_CORE_MAKE_FUNC(const OSSL_PARAM *, OP_keymgmt_settable_key_params, (void))
+
```

Comment:
For `OSSL_PKEY_PARAM_ECDH_COFACTOR_MODE`.

Reply:
Also, I realized that with the change for `EVP_PKEY_{size,bits,security_bits}` support in providers, we now have on all keymgmts gettable parameters that are not listed anywhere.

Should we somehow make more prominent which parameters are implicitly always available?
Or should a `OSSL_FUNC_KEYMGMT_GETTABLE_KEY_PARAMS` be included in the dispatch table of each keymgmt?

Reply:
@levitte might want to answer this

Reply:
With #11006, I redefined the conditions for a valid KEYMGMT method that if one of "get_params" and "gettable_params" exists, the other one must exist as well.

Regarding cofactor mode, I see it as something similar to PSS params for RSA, it's a value to be imported.  With #11006, that fulfills exactly the same thing as what you define above, since it allows importing stuff into already existing keys.